### ISSUE TITLE
KeyError: <gast.gast.Num object... when declaring function with default arguments

### ISSUE THREAD

**Issue by jeanlaroche** [serge-sans-paille/pythran#1020]
I'm getting a "KeyError: <gast.gast.Num object at 0x112b02510>" error on the following code.
```
import numpy as np

def test(signal,sR):
    N = 30
    F = 0.
    F2 = 22000
    FF = 10
    W = test2(sR, FF, N, F, F2)
    return W

def test2(sr,N,M=128,F=0.0,F2=0,B=False,No=1): #<- This breaks
#def test2(sr, N, M=128, F=0.0, F2=0):  # <- This works
    W = np.zeros(10)
    return W

```

**Comment by serge-sans-paille** [comment]
I can reproduce, keep in touch :-)

**Comment by serge-sans-paille** [comment]
Fixed by #1021

**Comment by jeanlaroche** [comment]
Confirmed fixed.
